x/bounty/keeper/msg_server.go

```diff
@@ -0,0 +1,68 @@
+package keeper
+
+import (
+	"context"
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/shentufoundation/shentu/v2/x/bounty/types"
+)
+
+type msgServer struct {
+	Keeper
+}
+
+// NewMsgServerImpl returns an implementation of the bank MsgServer interface
+// for the provided Keeper.
+func NewMsgServerImpl(keeper Keeper) types.MsgServer {
+	return &msgServer{Keeper: keeper}
+}
+
+var _ types.MsgServer = msgServer{}
+
+func (k msgServer) CreateProgram(goCtx context.Context, msg *types.MsgCreateProgram) (*types.MsgCreateProgramResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	nextID := k.GetNextProgramID(ctx)
```

Comment:
The `program_id` is not initialized at startup, such as being set to 1 for the first time?
I think the current value is 0.

Reply:
If it runs first before the initial NextProgramIDKey data, it will panic. So it needs to be initialized.